### PR TITLE
repo: Remove Vulkan-Layer-Factory

### PR DESCRIPTION
**Pull request by charles-lunarg**
The Vulkan-Layer-Factory project is being moved into its own repo for archival purposes. This commit removes it from this repo as well as documents where the archival repo is.

Repo is available here: https://github.com/LunarG/Vulkan-Layer-Factory

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,12 +25,12 @@ git remote set-head origin -a
...
+| Build Status                                                                                                                                                       |
+| :----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [![Build Status](https://github.com/LunarG/VulkanTools/actions/workflows/build_windows.yml/badge.svg?branch=main)](https://github.com/LunarG/VulkanTools/actions)  |
+| [![Build Status](https://github.com/LunarG/VulkanTools/actions/workflows/build_linux.yml/badge.svg?branch=main)](https://github.com/LunarG/VulkanTools/actions)    |
 | [![Build Status](https://github.com/LunarG/VulkanTools/actions/workflows/build_linux_gn.yml/badge.svg?branch=main)](https://github.com/LunarG/VulkanTools/actions) |
-| [![Build Status](https://github.com/LunarG/VulkanTools/actions/workflows/build_android.yml/badge.svg?branch=main)](https://github.com/LunarG/VulkanTools/actions) |
+| [![Build Status](https://github.com/LunarG/VulkanTools/actions/workflows/build_android.yml/badge.svg?branch=main)](https://github.com/LunarG/VulkanTools/actions)  |
```
Comment (juan-lunarg) [inline README.md:33]: I'd just remove this entirely. We already did in several repos.

**Review by juan-lunarg — COMMENTED** [review]
There is mention of VLF in the vulkan validation layers (In particular the integration testing. Ideally that is cleaned up as well).

**Comment by charles-lunarg** [comment]
Internal CI is failing due to having hardcoded references to the VLF layers. Not sure what the path forward is here - need to determine just how much they are used in internal CI.

File: README.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ VkTrace and VkReplay have been deprecated and replaced by [gfxreconstruct](https
 Both VkTrace and VkReplay have been removed from VulkanTools and can now be found in the [vktrace](https://github.com/LunarG/vktrace) archive.
 Both these tools are also no longer part of the [Vulkan SDK](https://www.lunarg.com/vulkan-sdk/).
 
+Vulkan-Layer-Factory is deprecated and has been archived in its own repository which is available [here](https://github.com/LunarG/Vulkan-Layer-Factory).
+
```
[the remark this answers is not in the document]
Comment (charles-lunarg) [reply README.md]: Done

**Review by juan-lunarg — APPROVED** [review]

**Comment by charles-lunarg** [comment]
Not sure why the ci bot didn't say anything, but the issue causing internal ci to fail has been resolved and there was a successful run of internal CI, so I'm going to go ahead and merge.

**Pull request merged**